db/db_impl.h

```diff
@@ -903,7 +906,7 @@ class DBImpl : public DB {
 
   WriteBufferManager* write_buffer_manager_;
 
-  WriteThreadImpl write_thread_;
+  std::unique_ptr<WriteThread> write_thread_;
```

Comment:
Can you explain why do you need this change? Pretty much the functions cannot be inlined with this change.